tags.lisp

```diff
@@ -141,7 +154,7 @@
   (declare (inline memq))
   (gethash attr *boolean-attributes*))
 
-(defvar *unvalidated-attribute-prefixes* '("data-" "aria-")
+(defvar *unvalidated-attribute-prefixes* '("data-" "aria-" "hx-")
```

Comment:
This is for htmx?

Reply:
yes. That would save every htmx user from having to specify `(pushnew "hx-" *unvalidated-attribute-prefixes* :test #’equal)`.
I'm of the mind that major frameworks should be supported out of the box. 
It's in the spirif of common lisp and it's pragmatic approach.

Reply:
This isn't the first time HTMX has been brought up specifically, so I'm OK with it.